packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -1,16 +1,100 @@
 import React, { useEffect } from 'react';
+import { translate } from '@aws-amplify/ui';
+import { FileUploaderProps } from './types';
+import { useFileUploader } from './hooks/useFileUploader';
+import { Text } from '../../../primitives';
+import { UploaderButton } from './UploaderButton';
 import { Previewer } from './Previewer';
 import { UploadDropZone } from './UploadDropZone';
-import { useFileUploader } from './hooks/useFileUploader';
 
-export function FileUploader(): JSX.Element {
-  const { setShowPreviewer } = useFileUploader();
+export function FileUploader({
+  acceptedFileTypes,
+  fileNames,
+  isPreviewerVisible,
+  level,
+  components: customComponents = {},
+  maxFiles,
+  maxMultipleSize,
+  maxSize,
+  multiple = true,
+  onChange,
+  onError,
+  onSuccess,
+  path,
+  variation = 'button',
+}: FileUploaderProps): JSX.Element {
+  const {
+    UploadDropZone = FileUploader.UploadDropZone,
+    UploaderButton = FileUploader.UploaderButton,
+  } = customComponents;
+  const {
+    setShowPreviewer,
+    showPreviewer,
+    inDropZone,
+    setFiles,
+    onDragStart,
+    onDragEnter,
+    onDragLeave,
+    onDrop,
+    onDragOver,
+  } = useFileUploader();
+
+  // eslint-disable-next-line no-console
+  console.log(
+    'todo:',
+    maxMultipleSize,
+    maxSize,
+    maxFiles,
+    onChange,
+    onError,
+    onSuccess,
+    path,
+    level,
+    fileNames
+  );
```

Comment:
Can we remove non-pertinent code from the PR please?

Reply:
ok